package.json

```
@@ -240,7 +240,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-push": "node ./bin/devkit-admin hooks/pre-push"
+      "commit-msg": "yarn -s ng-dev commit-message pre-commit-validate --file-env-variable HUSKY_GIT_PARAMS"
```

Adding this will allow you to automatically resume a commit message after a failed commit message attempt.



```suggestion
      "commit-msg": "yarn -s ng-dev commit-message pre-commit-validate --file-env-variable HUSKY_GIT_PARAMS",
      "prepare-commit-msg": "yarn -s ng-dev commit-message restore-commit-message-draft --file-env-variable HUSKY_GIT_PARAMS"
```